packages/trackerless-network/package.json

```diff
@@ -45,7 +46,6 @@
     "@types/uuid": "^10.0.0",
     "@types/yallist": "^4.0.1",
     "expect": "^29.6.2",
-    "ts-essentials": "^10.0.2",
```

Comment:
Is this change necessary?

Reply:
With the current approach yes, as we use it here:
```
export type ContentDeliveryLayerNeighborInfo = MarkRequired<ContentDeliveryLayerNeighborInfo_, 'peerDescriptor'>
```

In the `sdk` we use it similarly, and there it is a dependency, too.